pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/content/VelocityUtil.java

```diff
@@ -92,7 +95,7 @@ public class VelocityUtil {
     public static List<String> extractVelocityVariablesFromTemplate(String templateText) {
         try {
             RuntimeInstance ri = new RuntimeInstance();
-            SimpleNode node = ri.parse(templateText, TEMPLATE_NAME);
+            SimpleNode node = ri.parse(new StringReader(templateText), new org.apache.velocity.Template());
```

Comment:
I had a few different approaches here that I tried before this--thankfully the existing tests told me that my previous attempts (using `Template.getTemplate()`) were defective.  Props to `VelocityUtilTest`!